packages/naksha-gmaps-draw/package.json

```diff
@@ -34,6 +34,12 @@
   },
   "dependencies": {
     "@biodiv-platform/naksha-commons": "*",
-    "@turf/bbox": "^6.5.0"
+    "@chakra-ui/icons": "^2.1.1",
+    "@chakra-ui/react": "^2.8.2",
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
+    "@turf/bbox": "^6.5.0",
+    "framer-motion": "^10.16.16",
```

Comment:
is `framer-motion` a sub-dependency here?

Reply:
Yes